lib/keycloak-admin/client/token_client.rb

```diff
@@ -12,6 +12,12 @@ def token_url
       "#{realm_url}/protocol/openid-connect/token"
     end
 
+    def revoke_token_url(user_id)
+      puts "--------------- admin rest client ----------------"
```

Comment:
Could you remove the `puts` statements, please? The other parts of the code do not output such debugging statements.